tests/test_helpers.py

```diff
@@ -0,0 +1,96 @@
+"""Tests for test helpers."""
+
+from uuid import uuid4
+
+import pytest
+
+from .helpers import (
+    assert_statement_get_responses_are_equivalent,
+    string_is_date,
+    string_is_uuid,
+)
+
+
+def test_utils_string_is_date():
```

Comment:
```suggestion
def test_helpers_string_is_date():
```
Same suggestion for other functions in this file.